tests/openxml/wordprocessing/test_paragraph_properties.py

```diff
@@ -105,3 +105,21 @@ def test_returns_left_minus_hanging_ignoring_first_line(self):
         '''
         properties = self._load_from_xml(xml)
         self.assertEqual(properties.start_margin_position, 100)
+
+    def test_allow_decimal_indentation_for_hanging(self):
+        xml = '''
+            <pPr>
+                <ind left="123.0" hanging="23.0" />
+            </pPr>
+        '''
+        properties = self._load_from_xml(xml)
+        self.assertEqual(properties.start_margin_position, 100)
+
+    def test_allow_decimal_indentation_for_first_line(self):
+        xml = '''
+            <pPr>
+                <ind left="123.0" firstLine="50.0" />
```

Comment:
Is it worth having a test case where the decimal point is something other than zero? For example: 123.4.